packages/turf-polygon-tangents/index.js

```diff
@@ -0,0 +1,84 @@
+var coordEach = require('@turf/meta').coordEach;
+var helpers = require('@turf/helpers');
+var turfPoint = helpers.point;
+var turfFc = helpers.featureCollection;
+
+/**
+ * Finds the tangents of a {@link Polygon} from a {@link Point}.
+ *
+ * @name polygonTangents
+ * @param {Feature<Point>} point to calculate the tangent points from
+ * @param {Feature<Polygon>} polygon to get tangents from
+ * @returns {FeatureCollection|Feature<Point>} Feature Collection containing the two tangent points
+ * @example
+ * var poly = {
+ *   "type": "Feature",
+ *   "properties": {},
+ *   "geometry": {
+ *     "type": "Polygon",
+ *     "coordinates": [
+ *        [
+ *          [11, 0],
+ *          [22, 4],
+ *          [31, 0],
+ *          [31, 11],
+ *          [21, 15],
+ *          [11, 11],
+ *          [11, 0]
+ *        ]
+ *      ]
+ *   }
+ * }
+ * var point = {
+ *   "type": "Feature",
+ *   "properties": {},
+ *   "geometry": {
+ *     "type": "Point",
+ *     "coordinates": [61, 5]
+ *   }
+ * }
+ * var tangents = turf.polygonTangents(point, poly)
+ * //addToMap
+ * var addToMap = [tangents];
+ */
+module.exports = function (point, polygon) {
+    var eprev;
+    var enext;
+    var rtan = 0;
+    var ltan = 0;
+
+    coordEach(polygon, function (currentCoords, currentIndex) {
```

Comment:
You might need to look into using [`coordReduce`](https://github.com/Turfjs/turf/tree/master/packages/turf-meta#coordreduce), seems like this is the type of operation you are doing with `eprev` & `enext`.